source/src_lcao/LCAO_descriptor_old.cpp

```diff
@@ -2,18 +2,6 @@
 #ifdef __DEEPKS
```

Comment:
If no longer used, why should we keep it in repo?

Reply:
will clean up in the end; currently keeping this for checking purpose